build.sh

```diff
@@ -23,6 +23,22 @@ usage()
     exit 1
 }
 
+initDistroName()
+{
+    if [ "$__BuildOS" == "Linux" ]; then
+        # Detect Distro
+        if [ "$(cat /etc/*-release | grep -cim1 ubuntu)" -eq 1 ]; then
+            export __DistroName=ubuntu
+        elif [ "$(cat /etc/*-release | grep -cim1 centos)" -eq 1 ]; then
+            export __DistroName=centos
```

Comment:
At some point, we are going to need to extend these checks to bring in the version of the distro so we don't produce packages with an Ubuntu 14.04 RID when building on Ubuntu 15.10.  I wouldn't let that block progress here, but we should file an issue so we know to follow up.  I think @ericstj has a similar issue in CoreFX around building the "wrong" packages on Linux because today the build doesn't know about which Linux distro it is targeting and so we just produce all the packages.